developer/src/kmc-kmn/src/kmw-compiler/kmw-compiler.ts

```diff
@@ -140,6 +144,7 @@ export function WriteCompiledKeyboard(
     try {
       const data = callbacks.loadFile(sEmbedCSSFilename);
       sEmbedCSS = new TextDecoder().decode(data);
+      if(sEmbedCSS != '' && !sEmbedCSS.endsWith('\r\n')) sEmbedCSS += '\r\n';  // match CompileKeymanWeb.pas
```

Comment:
Wait... so it actually does emit `\r`... for CSS files, despite erasing / replacing them for HTML?

Reply:
The `\r\n` is replaced by the `RequotedString` function later... So I could have just added a space but this was a closer match to the existing code.